### PR TITLE
Update UserGuide.md

### PR DESCRIPTION
**Pull request by yyyaohhh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -229,23 +241,26 @@ Below shows what ModCraft should look like for executing the `add` command.
 
 ### Editing a module: `edit`
 
-Changes an attribute of a module **only if it exists**. Useful if you want to update information about a module or made a mistake while adding a module.
+Changes an attribute of a module **only if it exists**. Useful if you want to update information about a module or has made a mistake while adding a module.
```
Comment (rocketninja7) [inline docs/UserGuide.md:244]: have

File: docs/UserGuide.md [hunk 2]
```diff
@@ -153,28 +155,33 @@ Examples:
...
 
 <div style="display:flex; justify-content:space-around; align-items:center;">
   <img src="images/ImagesForUG/ap-before.png" alt="Before executing add command" style="height:400px; margin:20px;">
   <img src="images/ImagesForUG/ap-after.png" alt="After executing add command" style="height:400px; margin:20px;">
 </div>
 
+<br>
 
-Below shows what ModCraft should look like for adding a module to the Special Term (ST). The module `CS3230` is being added to Year 1, special term 1.
+Below shows you the process for adding a module to the Special Term (ST) in ModCraft. The module `CS3230` is being added to Year 1, Special Term 1.
```
Comment (rocketninja7) [inline docs/UserGuide.md:184]: one of the Special Terms

File: docs/UserGuide.md [hunk 3]
```diff
@@ -153,28 +155,33 @@ Examples:
...
+* There is **only one** Advance Placement semester and whenever `y/0` is entered, **regardless of the semester** entered, the module will be added or edited to the Advance Placement Semester.
 
 * The list for the Advance Placement semester and Special Terms 1 and 2 (ST) will only be displayed when there is a module within them.
 
 </md></div>
 
-Below shows what ModCraft should look like for adding a module the Advance Placement semester. The module `CS1010` is being added to Year 0 (the Advance Placement semester).
+<br>
+
+Below shows you the process for adding a module the Advance Placement semester in ModCraft. The module `CS1010` is being added to Year 0 (the Advance Placement semester).
```
Comment (rocketninja7) [inline docs/UserGuide.md:175]: adding a module to the

**Review by rocketninja7 — COMMENTED** [review]
Other than minor grammatical errors, LGTM

**Comment by yyyaohhh** [comment]
Updated accordingly 👍

**Pull request merged**